GDWeave/Godot/BuilinType.cs

```diff
@@ -0,0 +1,31 @@
+namespace GDWeave.Godot;
+
+public enum BuiltinType : int
```

Comment:
This is just the VariantType enum - we can't rename it due to mod compatibility so I think we should just use that enum instead of making a copy.

Reply:
I see now didn't know it existed. Should probably split files by type makes it a lot easier to find things like this.